app/services/bulk_republisher.rb

```diff
@@ -78,8 +78,6 @@ def republish_all_documents_by_ids(ids)
 private
 
   def republish_all_by_non_editionable_type(content_type_klass)
-    content_type_klass.find_each do |record|
-      Whitehall::PublishingApi.bulk_republish_async(record)
-    end
+    content_type_klass.find_each(&:bulk_republish_to_publishing_api_async)
   end
```

Comment:
I'm curious about the decision to put/hide the conditional logic in the `PublishesToPublishingApi` module. Is it worthy of an explanation in the commit message (without wanting to make it super long!)?

Reply:
I added this to `PublishesToPublishingApi` to keep the logic in the same place as other similar methods performing that check, e.g. `republish_to_publishing_api_async` which is essentially the same method, just without the bulk flag.